.github/workflows/workflow-code-coverage-python.yml

```diff
@@ -0,0 +1,27 @@
+name: Python Code Coverage
+
+on:
+  workflow_call:
+
+jobs:
+  code-coverage-python:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.11'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install coverage pytest
+
+    - name: Run tests with coverage
+      run: |
+        coverage run -m pytest
+        coverage report --show-missing --fail-under=60
```

Comment:
You can't fail right away, you need to allow the developer to catch up to the organization minimum first. This should probably be configurable from the calling repository github-workflow and set initially to the initially measured coverage (to ensure it doesn't regress). 

you also need to start by testing this workflow within a particular repo (membrane-backend) before moving it to the shared github-workflows...

Reply:
it's been tested from the fastapi-example repo.

Reply:
https://github.com/ai-cfia/fastapi-example/actions/runs/6380549510

Reply:
1) we've standardized on flask (for now, we may change in the future)
2) we've standardized on unittest (for now)

so although it could be useful in the future (and you can leave it there for now as fastapi-example is our bench for trying alternatives)
 
You may want to work on this issue first: https://github.com/ai-cfia/flask-example/issues/1

could give you a better idea of what we expect the backend developer to deliver from end to end to serve the current slate of products.